redaxo/src/core/lib/console/setup/run.php

```diff
@@ -66,6 +66,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return $value;
         };
 
+        rex_setup::init();
```

Comment:
@gharlan @bloep wurde hier bewusst das setup nicht initialisiert im commando?

inspiriert von der `.tools/bin/setup.php` die in diesem PR hier gelöscht wird, hab ich diese zeile ergänzt und jetzt passen die screenshots auch...

Reply:
Hmm, ich denke, es war nicht bewusst

Reply:
ich vermute im regulären release ist es nicht nötig, weil dann css dateien etc. durch den release prozess schon richtig an ort und stelle sind.

aber wenn man nur vom source cloned, ist der prozess nicht gelaufen und daher fehlt es dann